python/test/t_LinearModelAnalysis_std.py

```diff
@@ -72,3 +73,18 @@
 result = algo.getResult()
 analysis = ot.LinearModelAnalysis(result)
 results = str(analysis)
+
+# Fix #1820
+ot.RandomGenerator.SetSeed(0)
+distribution = ot.Normal()
+sample = distribution.getSample(30)
+func = ot.SymbolicFunction('x', '2 * x + 1')
+firstSample = sample
+secondSample = func(sample) + ot.Normal().getSample(30)
+linear_model = ot.LinearModelAlgorithm(firstSample, secondSample)
+linear_result = linear_model.getResult()
+test_result = ot.LinearModelTest.LinearModelFisher(
+    firstSample, secondSample, linear_result)
+linear_model_analysis = ot.LinearModelAnalysis(linear_result)
+ott.assert_almost_equal(test_result.getStatistic(), linear_model_analysis.getFisherScore(), 1e-13, 0)
+ott.assert_almost_equal(test_result.getPValue(), linear_model_analysis.getFisherPValue(), 0, 0)
```

Comment:
so they have the same meaning ?

Reply:
yes.
For next release I will make `LinearModelAnalysis` rely on `LinearModelTest`